src/app/sign-in/page.jsx

```diff
@@ -0,0 +1,53 @@
+import Link from "next/link";
+import Button from "@mui/material/Button";
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+
+export default function SignIn() {
+  return (
+     <div className = "Sign In" style={{ backgroundColor: 'lightgrey', minHeight: '100vh', color: 'black', display: 'flex', flexDirection: 'column' }}>
+     <Box
+     sx={{
+          width: '450px',
```

Comment:
Instead of using internal css, you can create an object for all the styles and use it